src/Scorecard.js

```diff
@@ -0,0 +1,193 @@
+'use strict';
+
+class Scorecard {
```

Comment:
Make your methods/variables shorter _(and easy to read/remember)_ but always meaning of what it does.